pdns/rec_channel_rec.cc

```diff
@@ -939,6 +939,18 @@ static ProxyMappingStats_t* pleaseGetProxyMappingStats()
   return ret;
 }
 
+static RemoteLoggerStats_t* pleaseGetRemoteLoggerStats()
+{
+  auto ret = make_unique<RemoteLoggerStats_t>();
+
+  if (t_protobufServers) {
+    for (const auto& s : *t_protobufServers) {
```

Comment:
clang-tidy complains about a too short name for `s`. Perhaps `server` would be better here.